README.md

```diff
@@ -21,6 +21,49 @@ framework
 * [ReportPortalPortal](https://github.com/znsio/ReportPortalIntegration/blob/main/README.md)
```

Comment:
Update section title from:
## Documentation for integration with ReportPortal and Applitools

to:
## Configuring ReportPortal and Applitools

Reply:
Done